src/house.cpp

```diff
@@ -390,8 +390,6 @@ void AccessList::parseList(std::string_view list)
 			break;
 		}
 
-		boost::algorithm::trim(line);
```

Comment:
just a question about this, are you sure this should be moved to player only?
as far as I know trim trims all spaces from current line, which would be logical to do to avoid comparing whitespaces?

Reply:
wait you're right i misread this

Reply:
intent is to only use the toLower in the case where we might actually need it, which if anything might be nowhere actually, but definitely not in the first 3 cases here

Reply:
I dont think we need toLower anywhere, and it is not used anywhere, cause we compare without case sensitibity, trim on the other hand should stay, because you'll want to trim trailling whitespaces from stuff players write into the window, like " nekiro" should be trimmed to "nekiro"